### PR TITLE
top tags added to home

### LINKED ISSUE #147: Show Top Tags in Home

### PR DESCRIPTION
**Pull request by giocodes**
closes #147
Added top.directive to include categories on the homepage

### CODE REVIEW

**Comment by Abazhenov** [comment]
reviewing

File: browser/js/top/top.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
     <div class="center row col-sm-12">
     <h1>Categories</h1>
     <p>Please select one from this awesome categories</p>
+    {{test}}
```
Comment (Abazhenov) [inline browser/js/top/top.html]: needs to be removed

File: browser/js/top/top.js [hunk 2]
```diff
@@ -4,25 +4,30 @@ app.config(function($stateProvider) {
...
     });
 });
 
 /*Category Controller*/
 
-app.controller('TagCtrl', function($scope, toptags) {
+app.controller('TagCtrl', function($scope, TagFactory, toptags) {
     $scope.toptags = toptags;
+
+    $scope.test = 'i am here lalalalala'
```
Comment (Abazhenov) [inline browser/js/top/top.js]: needs to be removed

Comment (giocodes) [inline browser/js/top/top.js]: bitmoji

**Pull request merged**